.travis.yml

```diff
@@ -1,8 +1,16 @@
-sudo: false
+sudo: required
+dist: trusty
 language: node_js
+before_install:
+- sudo dpkg --add-architecture i386
+- sudo apt-get update
+- sudo apt-get install -y wine
+
+install:
```

Comment:
Does replace to`before_script:`?
https://docs.travis-ci.com/user/gui-and-headless-browsers/#Using-xvfb-to-Run-Tests-That-Require-a-GUI